src/harberger-lib/components/Info.re

```diff
@@ -190,8 +191,8 @@ let make = (~tokenId: TokenId.t) => {
              "Foreclosure date: "->restr
              <Rimble.Tooltip
                message={
-                 "This is the date the deposit will run out and the current owner will lose ownership of token#"
-                 ++ tokenId->TokenId.toString
+                 "This is the date the deposit ran out and the current owner will lose ownership of "
```

Comment:
"will run" or "runs" is better. ran out is past tense but later in the sentence we switch to present tense when we say 'owner will lose ownership'

Reply:
I'll make it all past tense. (since this will only show when a token is foreclosed)